tests/array.js

```diff
@@ -3,6 +3,12 @@ var expect = chai.expect;
 var jsonexport = require('../lib/index');
 
 describe('Array', () => {
+    it('options omitted', () => {
```

Comment:
Moved the test down a bit so the error would still be caught albeit with a more cryptic error message. Might I suggest using a more aggressive linter to avoid such bugs in the future?